doc/classes/Window.xml

```diff
@@ -118,12 +118,14 @@
 			<return type="Vector2i" />
 			<description>
 				Returns the window's position including its border.
+				[b]Note:[/b] If window is not [member visible], this method return the same value as [member position].
```

Comment:
It's better to avoid relying on the property name as part of the natural English sentence, as this is difficult to translate.
```suggestion
				[b]Note:[/b] If the window is not visible, this method returns the same value as [member position].
```
Alternatively, if it's better to be explicit about the property, it could be "If the window's [member visible] property is [code]false[/code], ..." or "If [member visible] is [code]false[/code], ...".